pkg/machines/actions/store-actions.js

```diff
@@ -113,12 +113,13 @@ export function undefineNetwork({ connectionName, id }) {
     };
 }
 
-export function undefineStoragePool({ connectionName, id }) {
+export function undefineStoragePool({ connectionName, id, transientOnly }) {
```

Comment:
undefining a transient object sounds weird to me. I thought the concept was "define/undefine == permanent config", and "start/destroy == runtime objects"?

Reply:
You are completely right, i meant to right persistentOnly, then I changed it, and didn't take a second look. Fixing and reposting together with the syntax issues.

Reply:
You 're completely right, undefining transient resources is not possible.
undefineStoragePool does not really undefine the storage however by calling some API. If you look in the implementation, it just removes the object from the redux state.

I understand that the naming might be very confusing though, didn't realize it before I got the review. Probably I should rename that undefineStoragePool function to removeStoragePool and similarly for all resources.